### ISSUE TITLE
Joining a channel with the same name as a channel on a different server doesn't work

### ISSUE THREAD

**Issue by cinderblock** [thelounge/thelounge#4268]
In trying to move myself over to Libera.chat from Freenode, I ran into some annoyances.

I looked for other issues that covered this bug but did not find any.

## Steps to Reproduce

1. Join two IRC servers
2. Join a channel (ex: `#test`) on server 1.
3. Join a channel with the same name on server 2.

## Expected Results

Joined both channels. Can see messages from each channel in independent TheLounge "tabs".

## Actual Results

The 2nd instance of a channel with name `#test` is not joined. The UI changes the currently "selected" channel to the one that was connected to first.

On some occasions, a Vue error bar appears.

### Versions

- _Node version:_ v16.3.0
- _Browser version:_ Chrome 91.0.4472.124
- _Device, operating system:_ Server: Ubuntu 20 Client: Windows 10
- _The Lounge version:_ v4.2.0

**Comment by brunnre8** [comment]
![Tl](https://user-images.githubusercontent.com/16165454/123880084-3a0e0b80-d942-11eb-81c2-a58ca666b520.jpeg)

Works for me, what's the exact error you get?
Can you reproduce this all the time?

**Comment by cinderblock** [comment]
Well, now I'm extra confused. I was able to reliably reproduce this many times on my browser rather reliably.

Testing again on a different machine, I *think* I was able to reproduce it once, but trying many times after that, was not able to reliably reproduce.

Now I'm back on the machine that was originally exhibiting issues. Now it is working as expected (the machine was restarted for Windows Update).

This may have been related to the Freenode servers' transition to their new network. I had a notion that it was related to channels that had been closed on Freenode, like `##linux`, but trying that more, I don't see the problem.

I definitely saw the red Vue error bar yesterday that related to this issue but I think I hid it while trying to select the error's text.

It could have also been due to my backend updating unattended and me not refreshing/loading the frontend. It was updated recently enough, according to my logs, that this could have been the culprit. It was weird because other channels worked just fine.

Anyway, sounds like this might have been some ephemeral error that I'm happy to be past. Feel free to close this as appropriate.
